postgresql-on-ubuntu/metadata.json

```diff
@@ -2,6 +2,6 @@
   "itemDisplayName": "PostgreSQL 9.3 on Ubuntu VMs",
   "description": "This template creates a PostgreSQL streaming-replication between a master and one or more slave servers each with 2 striped data disks. The database servers are deployed into a private-only subnet with one publicly accessible jumpbox VM in a DMZ subnet with public IP.",
   "summary": "PostgreSQL stream-replication with multiple slave servers and a publicly accessible jumpbox VM",
-  "githubUsername": "arsenvlad",
-  "dateUpdated": "2015-04-08"
+  "githubUsername": "trentmswanson",
```

Comment:
We decided to just change it back.  I honestly don't care who's name is on this and just want to finish up the template changes and get it in the hands of those that need it.